### PR TITLE
Issue #9246: Setup testing infrastructure for Pebble ACME CA.

### LINKED ISSUE #9246: acmeCA-2.0: Add Pebble infrastructure for ACME fat tests
Investigate using test CA provider Pebble for ACME fat tests.

#9017

### PR DESCRIPTION
**Pull request by jvanhill**
fixes #9246

### CODE REVIEW

**Comment by jvanhill** [comment]
#run-libby

**Comment by jvanhill** [comment]
#run-libby
#build

File: dev/com.ibm.ws.security.acme/resources/OSGI-INF/l10n/metatype.properties [hunk 1]
```diff
@@ -16,8 +16,50 @@
...
+#directoryURI=ACME server directory URI
+#directoryURI.desc=The URI to the ACME server's directory object.
 
-#configParm2=Configuration parameter 2 for ACME
-#configParm2.desc=Description for parameter 2
+#domain=Domain name
+#domain.desc=A domain name to request a certificate for.
+
+#validFor=Valid for
+#validFor.desc=The duration of time the certificate signing request will request the certificate to be valid for. The default is defined by the CA server.
```
Comment (dmuelle) [inline dev/com.ibm.ws.security.acme/resources/OSGI-INF/l10n/metatype.properties]: **change**
The duration of time the certificate signing request will request the certificate to be valid for. The default is defined by the CA server.
**to**
The duration of time that the certificate signing request specifies for the certificate to be valid. The default is defined by the CA server.

File: dev/com.ibm.ws.security.acme/resources/OSGI-INF/l10n/metatype.properties [hunk 2]
```diff
@@ -16,8 +16,50 @@
...
+#locality.desc=Locality name to use for the certificate.
+
+#organization=Organization
+#organization.desc=Organization name to use for the certificate.
+
+#organizationalUnit=Organizational unit
+#organizationalUnit.desc=Organizational unit name to use for the certificate.
+
+#acceptTermsOfService=Accept Terms of Service
+#acceptTermsOfService.desc=Indicates whether the user accepts the terms of service for the ACME CA. If true, the terms of service will be automatically accepted. If false, and the ACME CA issues terms of service, the request for a certificate will fail.
```
Comment (dmuelle) [inline dev/com.ibm.ws.security.acme/resources/OSGI-INF/l10n/metatype.properties]: **change**
If true, the terms of service will be automatically accepted. If false, and the ACME CA issues terms of service, the request for a certificate will fail. 
**to**
If true, the terms of service is automatically accepted. If false, and the ACME CA issues terms of service, the request for a certificate fails.

File: dev/com.ibm.ws.security.acme/resources/OSGI-INF/l10n/metatype.properties [hunk 3]
```diff
@@ -16,8 +16,50 @@
...
+#orderRetries.desc=The number of times to retry updating the order status before aborting the order.
+
+#orderRetryWait=Order retry wait
+#orderRetryWait.desc=The amount of time to wait before retrying to refresh the status of an order.
+
+#accountContact=Account contact
+#accountContact.desc=A contact URL the ACME server can use to contact the client for issues related this the ACME account.
+
+#accountKeyFile=Account key file
+#accountKeyFile.desc=A path to the file containing a key identifier for a registered account on the ACME CA server. If the file does not exist a new account will be registered with the ACME CA server and the associated key will be written to this file. This file should be backed up to maintain control of the account on the ACME CA server.
```
Comment (dmuelle) [inline dev/com.ibm.ws.security.acme/resources/OSGI-INF/l10n/metatype.properties]: **change**
If the file does not exist a new account will be registered with the ACME CA server and the associated key will be written to this file. This file should be backed up to maintain control of the account on the ACME CA server.

**to**
If the file does not exist a new account is registered with the ACME CA server and the associated key is written to this file. Back this file up to maintain control of the account on the ACME CA server.

File: dev/com.ibm.ws.security.acme/resources/OSGI-INF/l10n/metatype.properties [hunk 4]
```diff
@@ -16,8 +16,50 @@
...
+#orderRetryWait.desc=The amount of time to wait before retrying to refresh the status of an order.
+
+#accountContact=Account contact
+#accountContact.desc=A contact URL the ACME server can use to contact the client for issues related this the ACME account.
+
+#accountKeyFile=Account key file
+#accountKeyFile.desc=A path to the file containing a key identifier for a registered account on the ACME CA server. If the file does not exist a new account will be registered with the ACME CA server and the associated key will be written to this file. This file should be backed up to maintain control of the account on the ACME CA server. 
+
+#domainKeyFile=Domain key file
+#domainKeyFile.desc=A path to the file containing a key identifier for a domain. If the file does not exist a new key will be generated and written to this file. This file should be backed up to maintain control of the domain.
```
Comment (dmuelle) [inline dev/com.ibm.ws.security.acme/resources/OSGI-INF/l10n/metatype.properties]: **change**
If the file does not exist a new key will be generated and written to this file. This file should be backed up to maintain control of the domain. 
**to**
If the file does not exist, a new key is generated and written to this file. Back this file up to maintain control of the domain.

**Comment by andrea-pichler** [comment]
No review requested from L2

**Comment by jvanhill** [comment]
#run-libby
#build

File: dev/com.ibm.ws.security.acme/resources/OSGI-INF/l10n/metatype.properties (hunk 1 above)
Comment (jvanhill) [reply dev/com.ibm.ws.security.acme/resources/OSGI-INF/l10n/metatype.properties]: Made the requested changes.

File: dev/com.ibm.ws.security.acme/resources/OSGI-INF/l10n/metatype.properties (hunk 3 above)
Comment (jvanhill) [reply dev/com.ibm.ws.security.acme/resources/OSGI-INF/l10n/metatype.properties]: Made the requested changes.

File: dev/com.ibm.ws.security.acme/resources/OSGI-INF/l10n/metatype.properties (hunk 2 above)
Comment (jvanhill) [reply dev/com.ibm.ws.security.acme/resources/OSGI-INF/l10n/metatype.properties]: Made the requested changes.

File: dev/com.ibm.ws.security.acme/resources/OSGI-INF/l10n/metatype.properties (hunk 4 above)
Comment (jvanhill) [reply dev/com.ibm.ws.security.acme/resources/OSGI-INF/l10n/metatype.properties]: Made the requested changes.

File: dev/com.ibm.ws.security.acme/resources/OSGI-INF/l10n/metatype.properties [hunk 5]
```diff
@@ -16,8 +16,50 @@
...
+orderRetryWait.desc=The amount of time to wait before retrying to refresh the status of an order.
+
+accountContact=Account contact
+accountContact.desc=A contact URL the ACME server can use to contact the client for issues related this the ACME account.
+
+accountKeyFile=Account key file
+accountKeyFile.desc=A path to the file containing a key identifier for a registered account on the ACME CA server. If the file does not exist, a new account is registered with the ACME CA server and the associated key is written to this file. Back this file up to maintain control of the account on the ACME CA server. 
+
+domainKeyFile=Domain key file
+domainKeyFile.desc=A path to the file containing a key identifier for a domain. If the file does not exist, a new key will be generated and written to this file. Back this file up to maintain control of the domain.
```
Comment (dmuelle) [inline dev/com.ibm.ws.security.acme/resources/OSGI-INF/l10n/metatype.properties]: **change**
If the file does not exist, a new key will be generated and written to this file.
**to**
If the file does not exist, a new key is generated and written to this file.

Comment (jvanhill) [reply dev/com.ibm.ws.security.acme/resources/OSGI-INF/l10n/metatype.properties]: Made the change.

**Comment by jvanhill** [comment]
#run-libby

**Comment by jvanhill** [comment]
#build

**Comment by jvanhill** [comment]
@kristip17 - the builds completed. It looks like all the failures are due to selenium servers being down, which I believe is the reason that buildcon is 1. Please merge this when buildcon stabilized.

**Pull request merged**